### PR TITLE
devel/R-cran-callr:3.5.1

### PR DESCRIPTION
**Pull request by yzgyyang**
new port: devel/R-cran-callr: Call R from R

Approved by: lwhsu

### CODE REVIEW

File: patches/devel.R-cran-callr.3.5.1.diff [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+new file mode 100644
+index 000000000000..2ad375bb6b1b
+--- /dev/null
++++ b/devel/R-cran-callr/pkg-descr
+@@ -0,0 +1,5 @@
++It is sometimes useful to perform a computation in a separate R process,
++without affecting the current R process at all. This packages does exactly
++that.
++
++WWW: https://github.com/r-lib/callr#readme
```
Comment (lwhsu) [inline patches/devel.R-cran-callr.3.5.1.diff:67]: drop `#readme`?

**Review by lwhsu — APPROVED** [review]

File: patches/devel.R-cran-callr.3.5.1.diff (hunk 1 above)
Comment (yzgyyang) [reply patches/devel.R-cran-callr.3.5.1.diff:67]: It appears that a lot of the R libraries have #readme in their WWW of R package DESCRIPTION. I feel like we could just use the original WWW in this case if it is a tradition, and it doesn't really change anything.

Comment (lwhsu) [reply patches/devel.R-cran-callr.3.5.1.diff:67]: That's also fine for me.

Comment (yzgyyang) [reply patches/devel.R-cran-callr.3.5.1.diff:67]: Sounds good, thanks!

**Pull request merged**